Dockerfile

```diff
@@ -1,55 +1,24 @@
-FROM ubuntu:18.04
-LABEL maintainer="Mirage"
-ENV PY_VER "3.7.4"
-ENV PY_VER_MAJOR "3.7"
+FROM python:3.9-slim-buster
```

Comment:
Oh, didn't realize this was using a debian base.

Reply:
So thinking perhaps we can do this, let's duplicate the original dockerfile, and name it `Dockerfile.ubuntu` and then name this new version `Dockerfile.debian`, that way ubuntu can be used to match the Heroku env with source and the debian could provide a faster local dev cycle.

Reply:
Okie doke. I would use an Ubuntu base regardless but for whatever reason apt is messed up and prohibitively slow on their docker image.